cmake/CMakeLists.txt

```diff
@@ -346,10 +346,8 @@ if (onnxruntime_MINIMAL_BUILD)
   if (onnxruntime_EXTENDED_MINIMAL_BUILD)
     # enable EPs that compile kernels at runtime
     add_compile_definitions(ORT_EXTENDED_MINIMAL_BUILD)
-
-    if (onnxruntime_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
-      add_compile_definitions(ORT_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
-    endif()
+    # enable runtime optimizations. These are required for NNAPI.
```

Comment:
>enable runtime optimizations. These are required for NNAPI. [](http://example.com/codeflow?start=6&length=59)

so now runtime optimizations are enabled for all extended minimal builds?
if so, we should get rid of onnxruntime_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD (eventually, can add a TODO comment for now).
also the set of source files is affected by onnxruntime_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD

Reply:
added the TODO. Don't quite understand the latest statement "also the set of source files is affected by onnxruntime_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD" is this a question or a general statement that if we enable all optimizations by default then there will be an implication on binary size?